bootcamp-milestone-2/src/app/portfolio/page.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from "./page.module.css"
+import projects from '../portfolioData.ts';
+import PortfolioProject from '../components/portfolioProject';
+
+
+
+export default function Portfolio() {
+    const project_loaded = projects.map(project => (
+        <PortfolioProject
+        {...project}
+        key={project.slug}
+        ></PortfolioProject>
+    ))
+
+    return (
+        <div className={styles.catalog}>
+            <h1>Projects</h1>
+            {/* {projects.map(project => 
```

Comment:
This is the more common way to do some sort of element mapping. No need to have a separate function